h/static/styles/annotations.scss

```diff
@@ -162,3 +212,25 @@ privacy {
     color: $text-color;
   }
 }
+
+.more, .less {
+  color: rgba(189, 28, 43, 0.5);
+  background: #fff;
```

Comment:
Does this need a background? Won't that do odd things against the gradient behind it?